### DISCUSSION TITLE
✅ With the latest version 27.7 (1407) I can no longer connect to my OpenWrt 23.05.5 router

### DISCUSSION THREAD

**Discussion by logicito** [isontheline/pro.webssh.net#1216] [General]
I am using the same profile/user/password/port, everything is the same and it has been working for months. From macOS terminal, SSH Files (ShellFish), or ShellBean apps I can connect without any problem, it is just from WebSSH that it fails. I have deleted the profile to the OpenWrt server, created again, same result, restarted the app, restarted the OpenWrt router, same, it works from any other app, but the WebSSH app from macOS or iPadOS or iOS.

Any idea on how to fix it? Thanks

<img width="412" alt="image" src="https://github.com/user-attachments/assets/6d960fa1-081d-4b4d-b201-6ee14a71d650">

**Comment by isontheline** [comment]
Hello @logicito 👋

Latest WebSSH release have a stronger security policy enforcement.

Please follow this guide to re-enable weak algorithms : https://webssh.net/documentation/help/SSH/terrapin-attack/

Could you give it a try?
If you need any further help feel free to contact me at any time.

**Reply by logicito** [reply]
Thank you, it is working now, appreciate the quick help

**Reply by isontheline** [reply]
Thank you a lot @logicito for your feedback 🙏
